docs/containers-storage-get-layer-data.md

```diff
@@ -0,0 +1,21 @@
+## containers-storage-get-container-data 1 "December 2020"
```

Comment:
I'm a bit confused over exactly what data is being gathered.  Title of the PR says "BigData", functions about Get/Set LayerBigData, help above "data attached to a layer", and man page says "lookaside" data.

Can we be consistent through out?  I'd prefer "lookaside data" if that holds.

Reply:
it can be any data you want to record for a layer.  In particular, it is going to record the zstd:chunked manifest in https://github.com/containers/storage/pull/775

Reply:
OK, but my comment still holds.  It appears we're calling it three or four different things inside the code and on the man pages.  Can we change the variable names/function calls to refer to lookaside data?

Reply:
BigData is the name already used for containers and images.  I think a different name can be confusing.

Should I change the comments?

Reply:
@giuseppe I'm fine with whatever name you use, my main issue was you used 3 or more names for it.  So if you did this:

LayerBigData  -> keep as is
"data attached to a layer" -> "BigData attached to a layer"
"lookaside" data -> BigData  or "the layer's BigData"

I'd be good with that.

Reply:
I am still unsure.  LayerBigData is about the API while "data attached to a layer" and "lookaside data" are used in the documentation and code comments.  Using `BigData` to explain what the Layer BigData is seems confusing:

e.g. converting:
`containers-storage get-layer-data - Retrieve lookaside data for a layer`
to:
`containers-storage get-layer-data - Retrieve bigdata for a layer`

does not seem like an improvement

Reply:
I'll let you make the final decision @giuseppe .  Not a real biggy, I just think it's a bit confusing as is.  I'll trust your judgment on the final naming.

Reply:
I prever
 containers-storage get-layer-data - Retrieve lookaside data for a layer